tests/sentry/api/endpoints/test_organization_member_team_details.py

```diff
@@ -36,25 +36,35 @@ def manager(self):
         return self.create_member(organization=self.org, user=self.create_user(), role="manager")
 
     @fixture
-    def team_member(self):
+    def member_on_team(self):
         return self.create_member(
             organization=self.org, user=self.create_user(), role="member", teams=[self.team]
         )
 
     @fixture
-    def team_admin(self):
```

Comment:
I took the liberty of changing these names since "team admin", etc., is now ambiguous.